src/error.rs

```diff
@@ -510,13 +508,16 @@ pub struct BulkWriteFailure {
 
     /// The error that occurred on account of write concern failure.
     pub write_concern_error: Option<WriteConcernError>,
+
+    pub(crate) inserted_ids: HashMap<usize, Bson>,
```

Comment:
This is necessary to track how many writes were attempted in the event of an error in an unordered bulk write, but it's also something that users might find useful (RUST-260).